python/pyarrow/tests/test_io.py

```diff
@@ -126,6 +126,44 @@ def test_python_file_read():
         pa.PythonFile(StringIO(), mode='r')
 
 
+@pytest.mark.parametrize("nbytes", (0, 1, 5, 100))
+@pytest.mark.parametrize("file_offset", (0, 5, 100))
+def test_python_file_get_stream(nbytes, file_offset):
+
+    data = b'data1data2data3data4data5'
+
+    f = pa.PythonFile(BytesIO(data), mode='r')
+    stream = f.get_stream(file_offset=file_offset, nbytes=nbytes)
+
+    # Subsequent calls to 'read' should match behavior if same
+    # data passed to BytesIO where get_stream should handle if
+    # nbytes/file_offset results in no bytes b/c out of bounds.
+    start = min(file_offset, len(data))
+    end = min(file_offset + nbytes, len(data))
+    buf = BytesIO(data[start:end])
+
+    # read some chunks
+    assert stream.read(nbytes=4) == buf.read(4)
+    assert stream.read(nbytes=6) == buf.read(6)
+
+    # Read to end of each stream
+    assert stream.read() == buf.read()
+
+    # Try reading passed the stream
```

Comment:
I think there is a misspelling? (not a native English speaker though...)
```suggestion
    # Try reading past the stream
```

Reply:
...and I'm an aspiring one [it seems](https://www.dailywritingtips.com/passed-vs-past/) :sweat_smile: